src/Liip/RMT/Action/CommandAction.php

```diff
@@ -19,6 +19,12 @@
  */
 class CommandAction extends BaseAction
 {
+    /**
+     * Saves the command vars.
+     * @var array
+     */
+    protected $commandVars = [];
```

Comment:
Why do you need that class property? Why not doing everything into the execute() method?
For info, from the execute method, you can do

    Context::getParam('current-version')
    Context::getParam('new-version')

Reply:
Running `Context::getParam('current-version')` in the `execute()` method would break since it's not yet defined for instance when ran in the prerequisites.

Running `Context::get('version-persister')->getCurrentVersion()` do the trick but if ran let's say in the `prepareCommand()`, that will end up being the new released version and not the current one.

The only way found at this point to get the current version is to get it in the constructor but I might be missing something